package.json

```diff
@@ -86,9 +86,8 @@
     "husky": "^8.0.1",
     "jest": "^26.6.3",
     "jest-cli": "^26.6.3",
-    "json5": "^2.2.3",
     "next-bundle-analyzer": "^0.6.7",
-    "prettier": "^3.0.3",
+    "prettier": "^1.19.1",
```

Comment:
don't have the full context here but seems strange that we would be downgrading this version

Reply:
Fixed in latest, thanks for the catch!